README.md

```diff
@@ -69,11 +72,29 @@ You can also choose one of the presets provided by the ROS2 Gem to create a LiDA
 the **RUNTIME** requirements.*
 
 ## Setup
+
 1. **Clone the Gem's repository.**
     ```bash
     git clone https://github.com/RobotecAI/o3de-rgl-gem.git
     ```
-2. **Register the Gem.** \
+2. **Switch to the correct branch.** \
+   Depending on which version of O3DE and ROS2 gem you use this step may differ.
+   Below is a universal table of supported build combinations:
+
+   | o3de-rgl-gem | [o3de/o3de-extras](https://github.com/o3de/o3de-extras) | [Robotecai/o3de-extras](https://github.com/robotecai/o3de-extras) | [o3de](https://github.com/o3de/o3de) |
+   |--------------|---------------------------------------------------------|-------------------------------------------------------------------|--------------------------------------|
+   | development  | development                                             | N/A                                                               | development      |
+   | development  | development                                             | N/A                                                               | stabilization/2409      |
+   | development  | N/A                                                     | rgl/stable-dev                                                    | point-release/23103                  |
+   | main         | point-release/23103                                    | N/A                                                               | point-release/23103                  |
+
+   **_Example:_** _For O3DE and O3DE extras on `point-release/23103` branches you would use the `main` branch of the RGL
```

Comment:
It is not is stated in line 89.

Reply:
As above.

Reply:
I've noticed however that after review changes this example is useless and removed it.